ynh_install_python/ynh_install_python

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+# Need also the helper https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/ynh_handle_getopts_args/ynh_handle_getopts_args
+
+pyenv_install_dir="/opt/pyenv"
+# PYENV_ROOT is the directory of pyenv, it needs to be loaded as a environment variable.
+export PYENV_ROOT="$pyenv_install_dir"
+# Required dependencies
+pyenv_dependencies="build-essential libssl1.0-dev|libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git"
+
+# Install python version management
+#
+# [internal]
+#
+# usage: ynh_install_pyenv
+ynh_install_pyenv () {
+  echo "Installation of pyenv - python version management" >&2
+  # Build an app.src for pyenv
+  mkdir -p "../conf"
+  echo "SOURCE_URL=https://github.com/pyenv/pyenv/archive/v1.2.19.tar.gz
```

Comment:
```suggestion
  echo "SOURCE_URL=https://github.com/pyenv/pyenv/archive/v${pyenv_version}.tar.gz
```
Variable like in [nodejs helper](https://github.com/YunoHost/yunohost/blob/7bfe564aab2300172b6aad68656dddbdd8933a27/data/helpers.d/nodejs#L20)?